tests/services/opportunity-test.js

```diff
@@ -0,0 +1,83 @@
+import chai from 'chai';
+import sinon from 'sinon';
+import stubValue from '../fakedata.js';
+import opportunityManager from '../../managers/opportunity/index.js';
+const expect = chai.expect;
+import OpportunityService from '../../services/opportunity/index.js';
+
+describe('OpportunityService', function () {
+  describe('create', function () {
+    it('should create a new Opprtunity', async function () {
+      const OpportunityManager = new opportunityManager();
```

Comment:
We should follow one convention at all places, 
Let's keep the class name starting with an uppercase letter and object name starting with lowercase letter. 
So, it should be changed to `const opportunityManager = new OpportunityManager();`